core/internal/filetransfer/file_transfer_default.go

```diff
@@ -63,7 +63,7 @@ func (ft *DefaultFileTransfer) Upload(task *Task) error {
 	stat, err := file.Stat()
 	if err != nil {
 		return fmt.Errorf(
-			"file transfer: upload: error when stat-ing %s: %v",
+			"error when stat-ing %s: %v",
```

Comment:
removed the "file transfer: upload:" from the errors we return here because I added that to `CaptureError` in the `Task` implementations